### PR TITLE
User Story 60501: 101-ddos-protection-plan

### PR DESCRIPTION
**Pull request by TomArcherMsft**
Part of POC to test generating sample code and articles using OpenAI.

### CODE REVIEW

**Review by lonegunmanb — APPROVED** [review]
Thanks @TomArcherMsft for opening this pr! LGTM 🚀. @grayzu any suggestions?

File: quickstart/101-ddos-protection-plan/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- [azurerm_virtual_network](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/virtual_network)
+
+## Variables
+
+| Name | Description | Default |
+|-|-|-|
+| `resource_group_name_prefix` | Prefix of the resource group name that's combined with a random ID so name is unique in your Azure subscription. | rg |
+| `resource_group_location` | Location of the resource group. | eastus |
+| `ddos_protection_plan_name` | Specify a DDoS protection plan name. | null |
+| `virtual_network_name` | Specify a DDoS virtual network name. | null |
```
Comment (grayzu) [inline quickstart/101-ddos-protection-plan/readme.md]: These variables do not exist so should be removed from readme.

**Review by grayzu — CHANGES_REQUESTED** [review]
Thanks for the PR!

Minor comment regarding the readme but otherwise looks fantastic!

**Comment by TomArcherMsft** [comment]
@grayzu I've removed the readme file's reference to the unused variables.

**Comment by grayzu** [comment]
Running tests now. Assuming they go well, this is good to merge.

**Pull request merged**